Jenkinsfile

```diff
@@ -37,31 +37,17 @@ node() {
           sh "node ./bin/checkshrinkwrap.js"
         }
 
-        stage('ATH - Jenkins 2.7.3') {
-          sh "cd acceptance-tests && ./run.sh --no-selenium --settings='-s ${env.WORKSPACE}/settings.xml'"
+        stage('ATH - Jenkins 2.46.3') {
+          sh "cd acceptance-tests && ./run.sh -v=2.46.3 --no-selenium --settings='-s ${env.WORKSPACE}/settings.xml'"
           junit 'acceptance-tests/target/surefire-reports/*.xml'
           archive 'acceptance-tests/target/screenshots/*'
         }
+
         if (env.JOB_NAME =~ 'blueocean-weekly-ath') {
           stage('ATH - Jenkins 2.60.1') {
             sh "cd acceptance-tests && ./run.sh -v=2.69.1 --no-selenium --settings='-s ${env.WORKSPACE}/settings.xml'"
```

Comment:
@imeredith there was another PR where vivek corrected the -v here - as is 2.69.1 really the right one